text/0756-helper-default-manager.md

```diff
@@ -156,6 +176,68 @@ setHelperManager(() => DEFAULT_HELPER_MANAGER, Function.prototype);
  - to register this helper manager, it should occur during app boot so developers do not need to import anything to
    trigger the `setHelperManager` call
 
+**Update 2022-01-13, during implementation**
```

Comment:
I find this odd to be in it's own section here, why isn't this updating the detailed design just above (or at least _also_ updating the detailed design)?

Reply:
Because it made sense to me, and i wanted to progression of discovery to be known. I haven't been aware of any 'proper' way to do these kinds of updates 🙃